tldextract/__init__.py

```diff
@@ -1,6 +1,20 @@
 """Export tldextract's public interface."""
 
-import pkg_resources
+import os
+
+try:
+    import pkg_resources
+except ImportError:
+    class pkg_resources(object):  # pylint: disable=invalid-name
+
+        """Fake pkg_resources interface which falls back to getting resources
+        inside `tldextract`'s directory.
+        """
+        @classmethod
+        def resource_stream(cls, _, resource_name):
```

Comment:
Nice catch. We would need to define `get_distribution`. I don't know if there's a good way to get the version of the package without `pkg_resources`. Perhaps we could just add `ImportError` to the exception handler that catches `pkg_resources.DistributionNotFound` below?